File: Linear_search.js
const linearSearch=(arr,value)=>{
    for(let index of arr){
        if(arr[index]===value){
            return index
        }
    }
    return -1
}

const arr=[3,4,5,6,4,3,7]
let res=linearSearch(arr,10)
console.log(res)

File: Linear_Search.py
def linear_search(arr,value):
    size=len(arr)
    for index in range(0,size):
        if(arr[index]==value):
            return index
    return -1


arr=[3,4,5,6,4,3,7]
res=linear_search(arr,10)
print(res)